.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+repos:
+  - repo: https://github.com/pycqa/isort
+    rev: 5.8.0
+    hooks:
+      - id: isort
+        name: isort (python)
+      - id: isort
+        name: isort (cython)
+        types: [cython]
+      - id: isort
+        name: isort (pyi)
+        types: [pyi]
```

Comment:
why does this hook has so many id's?

Reply:
It's the officially sanctioned config from the isort project:
https://pycqa.github.io/isort/docs/configuration/pre-commit/

Admittedly, I haven't really investigated it. I had just assumed it was so that isort would handle other types of files in addition to `*.py`.